application/models/card.py

```diff
@@ -5,8 +5,8 @@ class Card(db.Model):
     __tablename__ = 'card'
     name = db.Column(db.String(50), primary_key=True)
     noPlayers = db.Column(db.Integer, primary_key=True)
-    age = db.Column(db.Integer, default=1)
-    type = db.Column(db.String(10))
+    age = db.Column(db.Integer, primary_key=True)
```

Comment:
Yeap but in SQL a single column and only be a primary key if it is unique I think?